src/bootstrap/builder.rs

```diff
@@ -303,15 +303,14 @@ impl StepDescription {
     }
 
     fn maybe_run(&self, builder: &Builder<'_>, pathsets: Vec<PathSet>) {
```

Comment:
```suggestion
    fn maybe_run(&self, builder: &Builder<'_>, mut pathsets: Vec<PathSet>) {
```
See next part

Reply:
Thank you for reviewing the code and will modify it as soon as possible.